tests/test_trace.py

```diff
@@ -524,6 +533,20 @@ def test_send_traces_sanity(wrapped_post):
     )
 
 
+@mock.patch('urllib3.PoolManager.request')
+def test_send_traces_unicode(wrapped_post):
+    trace = trace_factory.get_or_create_trace()
+    runner = UnicodeReturnValueEventMock()
+    trace.set_runner(runner)
+    trace_factory.send_traces()
+    wrapped_post.assert_called_with(
+        'POST',
+        'collector',
+        body=json.dumps(trace.to_dict(), ensure_ascii=False),
```

Comment:
What are you trying to test here?
Can we avoid using the same way as in the code to generate the trace body?

Reply:
Verify that a unicode-containing trace generates the call, with all data (specifically the unicode)